satellite.py

```diff
@@ -40,6 +40,14 @@
 import neal
 import sys
 from dwave.system import LeapHybridSampler
+# Pr(at least one event)=1−Pr(none of the events)
```

Comment:
Minor style nitpick: please add two blank lines ahead of this function definition (https://www.python.org/dev/peps/pep-0008/#blank-lines)
```suggestion


# Pr(at least one event)=1−Pr(none of the events)
```

Reply:
done